model/accounts_mgmt/v1/account_resource.model

```diff
@@ -27,6 +27,7 @@ resource Account {
 	}
 
 	method Delete {
+	    in DeleteAssociatedResources Boolean
```

Comment:
let's set a default like cluster_resource

Reply:
Set deault to false